packages/preact-query/src/__tests__/QueryClientProvider.test.tsx

```
@@ -1,6 +1,6 @@
 import { queryKey, sleep } from '@tanstack/query-test-utils'
 import { render } from '@testing-library/preact'
-import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest'
+import { afterEach, beforeEach, describe, expect, vi, it } from 'vitest'
```

ESLint

[error] Member 'it' of the import declaration should be sorted alphabetically.

(sort-imports)